### PR TITLE
rebooting-machines: Add instructions for rebooting AWS Cache machines

### PR DESCRIPTION
**Pull request by issyl0**
- These didn't have any special instructions and, bleary-eyed yesterday
  morning, I didn't remember that there was an intricate and tedious
  process across all eight machines.
- So that everyone else can have this knowledge to hand, document the
  process.
- Took the instructions from the "restarting router" long-running card
  we had when there was the router bug.
- Platform Health has a card to make there be the standard number of
  cache machines rather than the current eight, so soon we can change
  the number in step 2 and the reboots become _a little_ less tedious.

https://trello.com/c/QWiMOe9Q/1686-document-how-to-safely-reboot-cache-machines-in-aws

### CODE REVIEW

File: source/manual/rebooting-machines.html.md [hunk 1]
```diff
@@ -103,6 +105,24 @@ There is a Fabric task to schedule a machine for downtime in Nagios for
 > manual entry on [adding disks](/manual/adding-disks-in-vcloud.html) for
 > more info.
 
+## Rebooting Cache machines in AWS
+
+The `cache` machines run the `router` app which handles live user traffic.
+To safely reboot them without serving too many errors to users, we must
+remove them from the AWS load balancer target group before rebooting:
```
Comment (cbaines) [inline source/manual/rebooting-machines.html.md]: `group` should probably be plural, as there are more than one.

File: source/manual/rebooting-machines.html.md [hunk 2]
```diff
@@ -103,6 +105,24 @@ There is a Fabric task to schedule a machine for downtime in Nagios for
...
+
+1. Login to AWS Console. Switch to the prod environment (`gds aws govuk-production-<your-role> -l`)
```
Comment (cbaines) [inline source/manual/rebooting-machines.html.md]: I'd probably say switch to the "right" or "relevant" environment, as you'll need to be in Staging for rebooting machines in Staging.

File: source/manual/rebooting-machines.html.md [hunk 3]
```diff
@@ -103,6 +105,24 @@ There is a Fabric task to schedule a machine for downtime in Nagios for
...
+1. Find the [Instance ID](https://eu-west-1.console.aws.amazon.com/ec2/home?region=eu-west-1#Instances:sort=desc:launchTime) of the critical machine(s) (probably all 8 `blue-cache` machines)
+1. `ssh [ip].eu-west-1.compute.internal` (find this from the reboots required alert listing)
+1. Remove the machine from the following [Target Groups](https://eu-west-1.console.aws.amazon.com/ec2/home?region=eu-west-1#TargetGroups:sort=targetGroupName):
+   1. cache-assets-origin
+   1. cache-www
+   1. cache-www-origin
+1. Check traffic again (should only be Smokey healthchecks now).
```
Comment (cbaines) [inline source/manual/rebooting-machines.html.md]: Does step (1, well there all 1, the 3rd step) involve checking the traffic? (as I don't think it's mentioned prior to this point)

Comment (issyl0) [reply source/manual/rebooting-machines.html.md]: These were amended from the Trello card for the router restarts. No, the "again" should go.

**Comment by issyl0** [comment]
Thanks Chris! Applied all the suggested changes.

**Pull request merged**